pages/mod/_id/edit.vue

```diff
@@ -449,29 +454,35 @@ export default {
     async saveMod() {
       this.$nuxt.$loading.start()
 
+      const modChanges = getDifferences(this.savedMod, this.mod)
+
       try {
         const data = {
-          title: this.mod.title,
-          description: this.mod.description,
-          body: this.mod.body,
-          categories: this.mod.categories,
-          issues_url: this.mod.issues_url,
-          source_url: this.mod.source_url,
-          wiki_url: this.mod.wiki_url,
-          license_url: this.license_url,
-          discord_url: this.mod.discord_url,
-          license_id: this.license.short,
-          client_side: this.clientSideType.toLowerCase(),
-          server_side: this.serverSideType.toLowerCase(),
-          slug: this.mod.slug,
-          license: this.license.short,
+          ...({ title: modChanges.title } || {}),
+          ...({ description: modChanges.description } || {}),
+          ...({ body: modChanges.body } || {}),
+          ...({ categories: modChanges.categories } || {}),
+          ...({ issues_url: modChanges.issues_url } || {}),
+          ...({ source_url: modChanges.source_url } || {}),
+          ...({ wiki_url: modChanges.wiki_url } || {}),
+          ...({ license_url: modChanges.license_url } || {}),
+          ...({ discord_url: modChanges.discord_url } || {}),
+          ...({ license_id: modChanges.license_id } || {}),
+          ...({ client_side: modChanges.client_side } || {}),
+          ...({ server_side: modChanges.server_side } || {}),
+          ...({ slug: modChanges.slug } || {}),
+          ...(modChanges.license
+            ? { license: modChanges.license.short } || {}
+            : {}),
+          /*
```

Comment:
This is fine, but could you add a visual indicator that the user is not allowed to edit the body and details (ex. disable input)? It was overlooked previously but it would be a nice addition to the PR.

Reply:
Sure

Reply:
Done :)